### PR TITLE
OFL MIP model for the blog post

### PR DESCRIPTION
**Pull request by muellerd**

### CODE REVIEW

**Review by heshagh — COMMENTED** [review]
@muellerd I scanned through this, but I don't think I'm the best person to provide formal approval here. My main ask is for those who are familiar with this use case for Nextmv with others confirm that what we've done here is generalized.

**Comment by muellerd** [comment]
@larsbeck @davidrijsman If you have time today, it would be great if you could quickly check this code in order to determine whether this is general enough compared to the OFL model you created.

File: Order Fulfillment with MIP/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Knapsack is a classic combinatorial optimization problem. Given a collection of
+items with a value and weight, our objective is to maximize the total value
+without exceeding the weight capacity of the knapsack.
+
+The input defines a number of items which have an id to identify the item, a
+weight and a value. There is also a volume, which is currently unused in the
+template but can be used to extend the template to a multi-dimensional knapsack.
+Additionally there are two capacities, one for weight and one for volume (again,
+the latter one is not used in this template).
```
Comment (larsbeck) [inline Order Fulfillment with MIP/README.md]: I think we should remove any mentioning of Knapsack in the readme or remove it entirely if that is easier

**Review by larsbeck — CHANGES_REQUESTED** [review]
Only minor comments - lets remove Knapsack from here, otherwise looks good

File: Order Fulfillment with MIP/README.md (hunk 1 above)
Comment (muellerd) [reply Order Fulfillment with MIP/README.md]: I changed the readme and removed knapsack from it. The only place left, where knapsack is referenced, is the code comment you saw in the main.go file.

**Review by larsbeck — APPROVED** [review]

**Pull request merged**